proposals/templatetags/diff_tags.py

```diff
@@ -10,8 +10,10 @@ def zip_equalize_lists(a, b):
     A zip implementation which will not stop when reaching the end of the
     smallest list, but will append None's to the smaller list to fill the gap
     """
-    a = list(a)
-    b = list(b)
+
+    a = [None] if a is None else list(a)
+    b = [None] if b is None else list(b)
+    
```

Comment:
This goes horribly wrong if `a == None` and `b == []`. You'll get a return of `[[None, None]]`, which of course should be `[]`.

As I think you tried to solve the issue of getting a plain `None` instead of an iterable, I think this should be changed to
```python

    a = [] if a is None else list(a)
    b = [] if b is None else list(b)
```